src/App.js

```diff
@@ -20,19 +22,20 @@ class App extends Component {
   }
 
   addSong = (id) => {
-    console.log("test")
    const songToAdd = this.state.songBook.find(song => {
     return song.id === id
     })
     this.setState({mySongs: [...this.state.mySongs, songToAdd]})
   }
 
-  removeSong = () => {
-
+  removeSong = (id) => {
+    const editedSongList = this.state.mySongs.filter(song => {
+      return song.id !== id
+    })
+    this.setState({mySongs: editedSongList})
   }
 
   render () {
-    console.log(this.state)
```

Comment:
Nice removal of the console.logs,

state setup looks good and accurate compared to the examples we've seen in lecture